tests/Fixtures/TestBundle/Entity/Dummy.php

```diff
@@ -130,6 +140,13 @@ class Dummy
      */
     public $nameConverted;
 
+    /**
+     * @var bool
+     *
+     * @ORM\Column(type="boolean", nullable=true, name="existsd")
```

Comment:
EXISTS is an SQL keyword. But we should probably use a name that doesn't look like a typo.

`it_exists`? :rofl: